draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -400,15 +400,9 @@ required to make use of the collaborative signaling:
   * Inspect client-to-server encrypted payload by network elements.
 
 REQ-ISP-SCALE:
-: The metadata and any other related state information that a network element (router) has to
-maintain for each additional flow it handles should be very low
-or none.
-
-REQ-CLIENT-VALIDATION:
-: The network needs to ensure the signal is coming from the same
-user/client that is part of the 5-tuple flow.  This is to ensure no
-other application influences the priority of another application's
-flow from within the same host.
+: The metadata and other state information that a router has to
+maintain for each additional media flow it handles should be kept
```

Comment:
```suggestion
maintain for each additional flow it handles should be kept
```

Reply:
no need to know this is "media flow". Removing is consistent with the assumption in the draft that we don't care about the application type, etc.

Reply:
Agree, just "flow" is better. (same for similar changes below)